ui/src/components/visit-us/Faq.js

```diff
@@ -0,0 +1,170 @@
+            <AccordionIcon as={AiFillPlusCircle} w={7} h={7} />
+          </AccordionButton>
+        </h2>
+        <AccordionPanel pb={4}>
+          Everyone who pre-registers for Sunday Celebration should commit to
+          showing up, because every slot is an opportunity for someone else to
+          come join in worship. If there are any extenuating circumstances that
+          prevent those pre-registered from joining in person, please notify us
+          via email at hongkong@hmcc.net by the Friday (before) 11:59PM
+        </AccordionPanel>
+      </AccordionItem>
+
+      <AccordionItem border="1px solid #E2E8F0">
+        <h2>
+          <AccordionButton fontWeight={700} fontSize="lg" p={4}>
+            <Box flex="1" textAlign="left">
+              What do I wear to church?
+            </Box>
+            <AccordionIcon as={AiFillPlusCircle} w={7} h={7} />
+          </AccordionButton>
+        </h2>
+        <AccordionPanel pb={4}>
+          No dress code in particular! However, we would suggest bringing a
+          light/thin jacket in case you get cold in our venue.
+        </AccordionPanel>
+      </AccordionItem>
+
+      <AccordionItem border="1px solid #E2E8F0">
+        <h2>
+          <AccordionButton fontWeight={700} fontSize="lg" p={4}>
+            <Box flex="1" textAlign="left">
+              Is there a service available specifically for my kids?
+            </Box>
+            <AccordionIcon as={AiFillPlusCircle} w={7} h={7} />
+          </AccordionButton>
+        </h2>
+        <AccordionPanel pb={4}>
+          Building Blocks is HMCC’s Children’s Ministry, specifically catered
+          for kids ages 3 to 11 and with all learning needs, will take place
+          alongside of our Sunday Celebrations. The children will be able to
+          interact with one another and the teachers over a time of worship,
+          Bible story and activities. For more information, please visit the{" "}
+          <Link href="https://hongkong.hmcc.net/building-blocks/" color="blue">
+            Building Block’s page
+          </Link>
+        </AccordionPanel>
+      </AccordionItem>
+
+      <AccordionItem border="1px solid #E2E8F0">
+        <h2>
+          <AccordionButton fontWeight={700} fontSize="lg" p={4}>
+            <Box flex="1" textAlign="left">
+              Would in-person service still be available under extreme weather
+              conditions?
+            </Box>
+            <AccordionIcon as={AiFillPlusCircle} w={7} h={7} />
+          </AccordionButton>
+        </h2>
+        <AccordionPanel pb={4}>
+          T9 or T10 signals hoisted by the Hong Kong Observatory: Our in-person
+          gatherings will be postponed or canceled. Note that if the signal is
+          lowered to T8 before or at 8:30AM on Sunday morning, we still have
+          Sunday Celebration at 10AM as scheduled. Please check our website or
+          social media accounts for updates regarding the latest information or
```

Comment:
maybe link to our social media account?

Reply:
should be fine since they'll be in the footer. Otherwise this part can get too convoluted